### PR TITLE
Auto GOMAXPROCS

### PR DESCRIPTION
**Pull request by mmatczuk**

### CODE REVIEW

File: cmd/forwarder/run/run.go [hunk 1]
```diff
@@ -67,6 +69,9 @@ func (c *command) runE(cmd *cobra.Command, _ []string) (cmdErr error) {
 
 	var ep []forwarder.APIEndpoint
 
+	logger.Infof("Forwarder %s (%s)", version.Version, version.Commit)
+	logger.Debugf("resource limits: GOMAXPROCS=%d GOMEMLIMIT=%s", runtime.GOMAXPROCS(0), os.Getenv("GOMEMLIMIT"))
```
Comment (Choraden) [inline cmd/forwarder/run/run.go:73]: nit: I don't like this empty log `GOMEMLIMIT=`, even if it's just a debug mode.

**Comment by Choraden** [comment]
LGTM

File: cmd/forwarder/run/run.go (hunk 1 above)
Comment (mmatczuk) [reply cmd/forwarder/run/run.go:73]: Would you like it like `<unknown>`

Comment (Choraden) [reply cmd/forwarder/run/run.go:73]: Yes, please.

**Comment by mmatczuk** [comment]
I changed it so that it only prints GOMEMLIMIT when set.

**Pull request merged**